### ISSUE TITLE
Coal spawn too high, is overridden, leaves samples, creates frustration

### ISSUE THREAD

**Issue by cjg9** [Darkosto/SevTech-Ages#2159]
## Issue / Bug
NOTE: "coal samples" refers to the rock-like structures found on the surface to indicate the presence of a coal vein in the chunk below.

Coal ore is not within chunk that contains surface coal samples.

Thoery on why this happens and how to fix: https://www.reddit.com/r/feedthebeast/comments/8fx31c/sevtech_ages_coal_not_spawningdetected_by_dowsing/

## Expected Behavior

Coal should be present in chunks that contain surface samples. OR surface samples should not be present in chunks absent of coal.

## Possible Solution

From the theory above, reduce the Y-level of coal generation to 60 or less

## Steps to Reproduce (for bugs)

A. Find a place to live that is at Y-Level 64
B. Locate nearby coal surface samples Y-Level 64
C. Be godforsakenly unlucky enough that the coal vein "spawns" above y - level 60 and is overridden with default blocks/air. (twice btw)
D. Attempt to find coal ore via dowsing rod.
E.??? - find nothing after searching the whole chunk
F.Profit (off the fact that you just got annoyed enough to write up this issue and get it fixed so several others don't have to go though this.)

(I mean no sass, I am a little worked up, however)

## Context

See above, This occurred while trying to create the portal needed to get dark quartz and aquamarine, specifically the coal blocks.

## Logs

* Client/Server Log:
https://drive.google.com/file/d/1hpvZt4MAeby1KL4HF52_3tKau_kqzXzQ/view?usp=sharing
* Crash Log:

## Client Information

* Modpack Version: 3.0.6
* Java Version: 1.8.0_161
* Launcher Used:
Twitch
* Memory Allocated:
8192 MB

* Server/LAN/Single Player: LAN (host)
* Resourcepack Enabled?: No
* Optifine Installed?: No

**Comment by 3jlowes** [comment]
Coal is very frequent. Out of all the coal veins, I mined, very few of them were sticking out on the surface in the side of the mountain is "overridden" by air. This is nothing to be frustrated about. as for coal samples I believe the way it works is if a chunk is marked by geolysis for ore population that chunk is guaranteed to have at least 1 coal ore in it. Then the rest of the vein could be spread to nearby chunks depending on how it was generated. So its just the way the mod was made for ore spawning in multiple chunks that dont have samples above them.

**Comment by cjg9** [comment]
That would make sense if either of my first two "samples" had any (even a single) ore in them, but I scanned the entire chunk down to Y=0 and found nothing, so I was left confused and frustrated twice.

**Comment by BigIrukandji** [comment]
I just spent about 4 hours looking for coal with no success. I find samples on the surface, and then dig about a dozen holes within that chunk all the way down to bedrock. I have yet to find a single coal ore on this world...... Copper and Tin are plentiful. I'm not sure what to do at this point, since I cannot progress any further without coal.

**Comment by BigIrukandji** [comment]
OK, I finally found the first vein. I had to search a total of 6 chunks with samples on the surface to find it. The other thing about the rarity of coal is that I probably find 20-25 samples of other ores before finding coal.

**Comment by N00bSkyw4lk3r** [comment]
Hi there...
I´m not sure if this helps or not but I think I would share my experience of yesterday 05.06.

I generated a beautiful world in 3.0 (i think) but let it rest for a good while. Picked it back up after updating to 3.07 and then progressed to Age 1.
I had no trouble at all finding copper (even in previously generated chunks) and little finding tin (in new generated Chunks).
I found a coal sample in a previously generated chunk (3.0) with no coal in it. I went according to the book to y44. Due to the tech staging I was not able to sufficiently quarry that chunk even in creative. The surface with the samples were at appr. y80 (so above max. Spawn).

Frustrated I went for another search FAR of in a new generated chunk. With the same experience.
So I jumped back to creative and tested the dowsing rod and it worked just fine in all directions (also bove and below). So I dug down to y45 again and tested the dowsing rod for anything above (up to y93 then). No coal found. Frustrated and late the night I clicked that rod wildly and suddenly: "Coal: 6". it took me like 10 minutes to figure out were I got that message: It was below. so i dug even deeper and found the vein at y23 (the achievment speaks of y 22 and above I think). So I went back to the 1. deposit and found it at y20 (so even deeper than the achievment told).

TL;DR: The coal veins seem to generate BELOW min levels of book and achievment info. Maybe it is configured to exclude the given Y-levels instead of limiting to those?

Maybe this helps, else don´t mind ;)

Kr,
A n00b.
